src/include/ZividPython/Releasable.h

```diff
@@ -129,12 +129,13 @@ namespace ZividPython
     class Singleton
     {
     public:
-        Singleton()
+        template<typename... Args>
+        Singleton(Args &&...args)
```

Comment:
we need to delete the copy/move ctors or restrict the type of `Args` to not be `Singleton`, else this constructor will take precedence over copy/move.

Reply:
pushed a fixup for this